src/wazuh_modules/vulnerability_scanner/tests/unit/factoryOrchestrator_test.cpp

```diff
@@ -10,84 +10,172 @@
  */
 
 #include "factoryOrchestrator_test.hpp"
-#include "MockCleanAgentInventory.hpp"
-#include "MockDetailsAugmentation.hpp"
-#include "MockFetchAllGlobalDb.hpp"
-#include "MockInventorySync.hpp"
-#include "MockOsScanner.hpp"
-#include "MockPackageScanner.hpp"
-#include "MockQueryAllPkgsAgt.hpp"
-#include "MockResultIndexer.hpp"
-#include "MockSendReport.hpp"
+#include "MockDatabaseFeedManager.hpp"
+#include "MockIndexerConnector.hpp"
+
+constexpr int PACKAGE_SCANNER_ID = 0;
+constexpr int EVENT_PACKAGE_ALERT_DETAILS_BUILDER_ID = 1;
+constexpr int SCAN_OS_ALERT_DETAILS_BUILDER_ID = 2;
+constexpr int EVENT_DETAILS_BUILDER_ID = 3;
+constexpr int ALERT_CLEAR_BUILDER_ID = 4;
+constexpr int OS_SCANNER_ID = 5;
+constexpr int CLEAN_INVENTORY_ID = 6;
+constexpr int EVENT_DELETE_INVENTORY_ID = 7;
+constexpr int EVENT_INSERT_INVENTORY_ID = 8;
+constexpr int SCAN_INVENTORY_SYNC_ID = 9;
+constexpr int CLEAR_SEND_REPORT_ID = 10;
+constexpr int EVENT_SEND_REPORT_ID = 11;
+constexpr int RESULT_INDEXER_ID = 12;
+constexpr int FETCH_FROM_GLOBAL_DB_ID = 13;
+constexpr int CLEAN_AGENT_INVENTORY_ID = 14;
+constexpr int QUERY_PKG_ID = 15;
```

Comment:
Suggestion (Apply it if considered useful): This could be inside an enum class
```suggestion
enum class ScannerMockID : int {
    PACKAGE_SCANNER = 0,
    EVENT_PACKAGE_ALERT_DETAILS_BUILDER = 1,
    SCAN_OS_ALERT_DETAILS_BUILDER = 2,
    EVENT_DETAILS_BUILDER = 3,
    ALERT_CLEAR_BUILDER = 4,
    OS_SCANNER = 5,
    CLEAN_INVENTORY = 6,
    EVENT_DELETE_INVENTORY = 7,
    EVENT_INSERT_INVENTORY = 8,
    SCAN_INVENTORY_SYNC = 9,
    CLEAR_SEND_REPORT = 10,
    EVENT_SEND_REPORT = 11,
    RESULT_INDEXER = 12,
    FETCH_FROM_GLOBAL_DB = 13,
    CLEAN_AGENT_INVENTORY = 14,
    QUERY_PKG = 15
};
```